reference/3.0/Microsoft.PowerShell.Core/About/about_Aliases.md

```diff
@@ -194,8 +194,8 @@ Get-Help about_Functions
 
 PowerShell aliases are represented by objects that are instances of the
 System.Management.Automation.AliasInfo class. For more information about this
-type of object, see [AliasInfo Class](http://go.microsoft.com/fwlink/?LinkId=143644)
-in the Microsoft Developer Network (MSDN) library.
+type of object, see [AliasInfo Class][aliasinfo] in the Microsoft Developer
```

Comment:
Raw: 
```For more information about this type of object, see [AliasInfo Class][aliasinfo] ``` 
this doesn't look like a valid link, but it renders correctly in GitHub? Am I misunderstanding the link syntax here?